Documentation/helm-operator.md

```diff
@@ -100,6 +100,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `logLevel`                   | Global log level                                                                                        | `INFO`                                                 |
 | `nodeSelector`               | Kubernetes `nodeSelector` to add to the Deployment.                                                     | <none>                                                 |
 | `tolerations`                | List of Kubernetes `tolerations` to add to the Deployment.                                              | `[]`                                                   |
+| `rookCurrentNamespaceOnly`   | Whether the operator should watch cluster CRD in its own namespace or not                               | `true`                                                 |
```

Comment:
In the helm chart, Rook is implied, so I think this can just be `currentNamespaceOnly`.